redirections.yml

```diff
@@ -1,4 +1,46 @@
 "301":
+  -
+    old: "/languages/ruby/private-gem-from-git-on-scalingo/"
+    new: "/languages/ruby/private-gem-from-git-on-scalingo"
+  -
+    old: "/platform/app/transfer-ownership"
+    new: "/platform/app/lifecycle-management#transfer-ownership-of-an-application"
```

Comment:
Invalid destination:

https://documentation-service-pr279.scalingo.io/platform/app/lifecycle-management#transfer-ownership-of-an-application

Reply:
Fixed in previous push